package-lock.json

```diff
@@ -193,110 +193,6 @@
         }
       }
     },
-    "@evocateur/libnpmaccess": {
```

Comment:
There are way too many removals from this lockfile.

Reply:
I don't know what the removals are.  Probably someone can `git checkout origin/master -- package-lock.json && npm install` to clean that up.

Reply:
In my experience it's usually because of a manual addition to a root package.json that didn't already have a `node_modules` installed. Or possibly the other way around. Anyway, fairly easy to fix.